scripts/stage-release.sh

```diff
@@ -169,7 +162,17 @@ then
     exit
 fi
 
+# create GitHub release via CLI https://cli.github.com/manual/gh_release_create
+gh release create "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md" --draft
```

Comment:
To make this work locally I had to replace `gh` with a variable so I could inject a custom command (the one password plug-in / wrapper system uses aliases which don't work in scripts). 

I was going to open a PR doing that then I spotted this one, so I guess I'm asking if its worth doing so we/I can still run it locally or are we close enough that we can do it all via actions and thats now moot?

Reply:
Personally, I think we should be running releases via actions exclusively if we can. This should eliminate the risk of accidentally releasing the wrong branch/WIP code/etc, and ensure that releases are run exactly the same way every time regardless of who is running them.

However I'm also not opposed to having the script do some command discovery or allow injecting custom commands if it's not too much work to add it.